cartographer/mapping/2d/grid_2d.h

```diff
@@ -42,6 +42,7 @@ class Grid2D : public GridInterface {
          ValueConversionTables* conversion_tables);
   explicit Grid2D(const proto::Grid2D& proto,
                   ValueConversionTables* conversion_tables);
+  virtual ~Grid2D() {}
```

Comment:
this is redundant, since the parent destructor is virtual already. If you want to make it explicit here you may instead use:
`~Grid2D() override = default` (I always like this since it kind of provides a sanity check that the parents destructor is virtual)

Reply:
+1. Please remove `virtual`, but add `override`, see https://google.github.io/styleguide/cppguide.html#Inheritance. Similar below.